include/uapi/sound/sof/tokens.h

```diff
@@ -191,4 +192,9 @@
 /* COPIER */
 #define SOF_TKN_INTEL_COPIER_NODE_TYPE		1980
 
+/* pin_binding */
+#define SOF_TKN_PIN_BINDING_BIND_TYPE		2000
+#define SOF_TKN_PIN_BINDING_QUEUE_ID		2001
```

Comment:
We bind a source queue to a sink queue. What does 'queue_ID' mean here. Is this referring to the source, the sink, something else?